doc/api.apib

```diff
@@ -53,6 +53,67 @@ Resources related to the Workflows registered in Styx.
               }
         }
 
+
+### Delete Workflow [DELETE]
+
+Returns 200 OK if the Workflow was found an deleted, 404 Not Found otherwise.
+
++ Response 200
+
++ Response 404
+
+## Workflow Configuration [/v2/workflows/{component_id}]
+
+Accepts a Workflow Configuration update as a JSON object. Returns 200 OK if the Workflow was created or updated with the supplied configuration.
+
++ Parameters
+    + component_id: `styx-canary` (string) - Workflow Component
+
+### Create or update Workflow [POST]
+
++ Request (application/json)
+
+        {
+          "id": "StyxCanary",
+          "schedule": "days",
+          "offset": null,
+          "docker_image": null,
```

Comment:
Updated the API to only accept non-null values for docker_image https://github.com/spotify/styx/pull/243/commits/7e068b2e79dcbd847051cd864794f176768bb72c